linkedlist.c

```diff
@@ -0,0 +1,61 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct node
+{
+int data;
+struct node *next;
+};
+
+struct node *head=NULL;
+
+
+void insertion_beginning()
+{
+struct node *newNode;
+newNode =NULL;
+newNode=(struct node*)malloc(sizeof(struct node));
+printf("enter data :");
+scanf("%d",&newNode->data);
+if(head==NULL)
+{
+head=newNode;
+}
+else
+{
+newNode->next=head;
+head=newNode;

```

Comment:
Is this code to keep inserting elements infront/beginning of the list?
Can you share the question please or update that in commit message?